data/kernels/blendop.cl

```diff
@@ -1009,9 +1009,9 @@ blendop_mask_tone_curve(__read_only image2d_t mask_in, __write_only image2d_t ma
   float opacity = read_imagef(mask_in, sampleri, (int2)(x, y)).x;
   float scaled_opacity = (2.f * opacity / gopacity - 1.f);
   if (1.f - brightness <= 0.f)
-    scaled_opacity = opacity <= FLT_EPSILON ? -1.f : 1.f;
+    scaled_opacity = opacity <= 16 * FLT_EPSILON ? -1.f : 1.f;
```

Comment:
Can you tell me more about the choice of 16 here ?

Reply:
The factor 16 is basically an empirical safety factor. I have no better justification so far. :-(

Reply:
I would add that as a comment:

`// 16 is just an empirical safety factor`

or add a `#define` for it. Names are better than magic numbers :-)